ui/apps/platform/cypress/integration/vulnerabilities/nodeCves/cveDetailPage.test.ts

```diff
@@ -0,0 +1,73 @@
+import withAuth from '../../../helpers/basicAuth';
+import { hasFeatureFlag } from '../../../helpers/features';
+
+describe('Node CVEs - CVE Detail Page', () => {
+    withAuth();
+
+    before(function () {
+        if (!hasFeatureFlag('ROX_VULN_MGMT_NODE_PLATFORM_CVES')) {
+            this.skip();
+        }
+    });
+
+    it('should restrict access to users with insufficient permissions', () => {
+        // check that users without Node access cannot access the Node CVE Detail page directly
+    });
+
+    it('should only show relevant filters for the Node CVE Detail page', () => {
+        // check the advanced filters and ensure only the relevant filters are displayed
+    });
+
+    it('should link to the correct pages', () => {
+        // clicking the Node CVEs breadcrumb should navigate to the overview page with the CVE tab selected
+        // clicking a Node name in the list should navigate to the correct Node details page
```

Comment:
If these two have separate starting points, then two tests might be more reliable. Multi-step tests of classic side panel containers have had intermittent failures solved only by intercepts in the helper functions. Which I strongly recommend, but that is a separate subject.

Reply:
Good call, I'll split this test.